### DISCUSSION TITLE
I see A32NX in content manager in not installed folder.

### DISCUSSION THREAD

**Discussion by Five5Head** [flybywiresim/aircraft#4833] [Help]
As the title says, I see A32NX in a NOT INSTALLED folder, is there anything I can do about it?
I have ms flight sim from the MS store.
![2021-05-16_LI](https://user-images.githubusercontent.com/71512111/118404471-4b8da380-b673-11eb-9da3-febf6206b029.jpg)

**Comment by ExampleWasTaken** [comment]
> 
> 
> As the title says, I see A32NX in a NOT INSTALLED folder, is there anything I can do about it?
> I have ms flight sim from the MS store.
> ![2021-05-16_LI](https://user-images.githubusercontent.com/71512111/118404471-4b8da380-b673-11eb-9da3-febf6206b029.jpg)

That doesn't matter. It's a bug in the content manager. As long as you see the aircraft in your aircraft selection everything is fine.
